FuncGameDev/FSharp/Unity/Update.fs

```diff
@@ -15,6 +15,7 @@ type Updater() =
         ()
 
     member this.Update() =
+        CameraUpdater.update
```

Comment:
Move this later. We should update the gamestate before we update any GameObjects.